core-services/prow/02_config/_config.yaml

```diff
@@ -179,14 +179,23 @@ plank:
         entrypoint: gcr.io/k8s-prow/entrypoint:v20210730-987d7118e9
         initupload: gcr.io/k8s-prow/initupload:v20210730-987d7118e9
         sidecar: gcr.io/k8s-prow/sidecar:v20210730-987d7118e9
-    openshift-priv:
+  - cluster: arm[0-9]{2}
+    config:
+      utility_images:
+        clonerefs: gcr.io/k8s-prow/v20210730-987d7118e9-arm64
+        entrypoint: gcr.io/k8s-prow/entrypoint:v20210730-987d7118e9-arm64
+        initupload: gcr.io/k8s-prow/initupload:v20210730-987d7118e9-arm64
+        sidecar: gcr.io/k8s-prow/sidecar:v20210730-987d7118e9-arm64
+  - config:
       gcs_configuration:
         bucket: origin-ci-private
         mediaTypes:
           log: text/plain
       gcs_credentials_secret: gce-sa-credentials-gcs-private
-    openshift/osde2e:
+    repo: openshift-priv
```

Comment:
This is a github org, not repo

Reply:
it's a catch-all field, see https://github.com/kubernetes/test-infra/tree/master/prow/plank